charts/cadvisor/values.yaml

```diff
@@ -7,9 +7,9 @@ image:
   ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
   ##
   pullSecrets: []
-
 container:
   port: 8080
+  hostPort: 8080
```

Comment:
@wesuuu Could you please add the "hostIP" field here as well and comment both that they are not set by default.